### DISCUSSION TITLE
Does v1 still get security patches?

### DISCUSSION THREAD

**Discussion by cenkalti** [pallets/flask#5267] [Q&A]
Is it safe to keep using Flask v1 and not upgrade to v2?

**Comment by ThiefMaster** [comment]
**It's general not safe to use outdated/unmaintained software versions.**

If we found a *really* critical issue we might at least consider backporting it, but it is not safe to expect or assume so --> You should absolutely update.

Providing free "LTS-style" support for old versions is not something particularly sustainable for open source projects.
